templates/taxi/manufacturer_list.html

```diff
@@ -10,19 +12,27 @@ <h1>Manufacturer List
           <th>ID</th>
           <th>Name</th>
           <th>Country</th>
+          <th>Update</th>
+          <th>Delete</th>
         </tr>
 
         {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
+          <tr>
+            <td>
+                {{ manufacturer.id }}
+            </td>
+            <td>
+                {{ manufacturer.name }}
+            </td>
+            <td>
+                {{ manufacturer.country }}
+            </td>
+            <td>
+              <a href="{% url "taxi:manufacturer-update" pk=manufacturer.id %}"></a>
+            </td>
+            <td>
+              <a href="{% url "taxi:manufacturer-delete" pk=manufacturer.id %}"></a>
+            </td>
```

Comment:
Add something to the `a` tag, because you don't have any clickable item on the webpage